_aggregations/bucket/auto-interval-date-histogram.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: default
+title: Auto-date histogram
+parent: Bucket aggregations
+grand_parent: Aggregations
+nav_order: 15
+---
+
+# Auto-date histogram
+
+The `auto_date_histogram` aggregation automatically creates date histogram buckets based on the time range of your data. This aggregation is particularly useful when you are working with time-series data and want to visualize or analyze data over different time intervals without having to specify the bucket size manually.
+
+To use the `auto_date_histogram` aggregation, you need to specify the field containing the date or timestamp values. Additionally, you can provide optional parameters to customize the behavior of the aggregation.
+
+For example, you can use the `auto_date_histogram` aggregation to create date-based buckets:
+
+```json
+GET /sample-index/_search
+{
+  "size": 0,
+  "aggs": {
+    "histogram": {
+      "auto_date_histogram": {
+        "field": "timestamp",
+        "buckets": 2,
+        "minimum_interval": "day",
+        "time_zone": "America/Los_Angeles",
```

Comment:
What are the possible formats for `time_zone`?

Reply:
@bowenlan-amzn Please help me address the information gap. See comment above.